draft-ietf-tls-esni.md

```diff
@@ -930,6 +959,28 @@ in-band when possible, such as through the use of OCSP stapling, and clients
 SHOULD take steps to minimize or protect such requests during certificate
 validation.
 
+## Abuse of ECH Acceptance Signal
```

Comment:
Why is this Abuse? "ECH Acceptance Signal Collisions" seems to better capture the text below.

Reply:
The title is meant to encompass all possible attacks that exploit the acceptance confirmation signal. How about "Attacks Exploiting Acceptance Confirmation"?

Reply:
```suggestion
## Attacks Exploiting Acceptance Confirmation
```